dev-requirements.txt

```diff
@@ -3,7 +3,7 @@ behaving==2.0.0
 Appium-Python-Client<=0.52
 ckanapi==4.3
 ckantoolkit>=0.0.4
-factory-boy
+factory-boy==2.12.0
```

Comment:
This will break CKAN 2.8 compatibility. The OKFN containers already provide a `factory-boy` version suitable for the target CKAN, we don't need to override it.

Reply:
I guess CKAN uses this version to guarantee the same result in both python2 and python3. If you will try to install it without specifying the version, you will have different versions in py2 & py3

Reply:
Having different factory-boy versions in py2 and py3 is fine, so long as our tests can handle it. The version needs to match CKAN's expectations, though. 2.12.0 generally won't work on CKAN 2.8.

Reply:
Will be fixed and pushed by Alex shortly.